### DISCUSSION TITLE
Which managed extensions you guys have

### DISCUSSION THREAD

**Discussion by Luigi-Master** [3kh0/ext-remover#1300] [Polls JOIN TITANIUM NETWORK TO SHARE PLEASE]
I'm pretty curious of what extensions you have that block you, I have securely that blocks me. I also have go guardian, technically also gopher buddy but that doesn't do anything to me.

**Comment by schoolexploitkid** [comment]
i also have netsupport

**Reply by Luigi-Master** [reply]
Interesting, I have never heard of that extension.

**Reply by schoolexploitkid** [reply]
and smth that blocks crosh idk which one it is.

**Reply by Luigi-Master** [reply]
oh probably a managed network or school policies

**Reply by Perplexedykwhat** [reply]
isn't netsupport basically so teachers/staff can see your screen? I thought its for watching a students screen and monitoring them. I also have netsupport with lightspeed though

**Reply by S-PScripts** [reply]
Yeah our school has claasroomcloud(owned by net support) on our Chromebooks and net support on the school computers

**Reply by S-PScripts** [reply]
> isn't netsupport basically so teachers/staff can see your screen? I thought its for watching a students screen and monitoring them. I also have netsupport with lightspeed though

Yeah it is (it sucks)

**Reply by schoolexploitkid** [reply]
our teachers dont even use it

**Reply by Luigi-Master** [reply]
My district has go guardian installed but they don't use it for some reason? It's really rare to see them use it in my school.

**Comment by py660** [comment]
why are there so many new polls/discussions, just use the preexisting one!

**Reply by Luigi-Master** [reply]
I really didn't know there was a pre-existing one, I know it would be a pretty obvious poll question but scrolling through discussions is pretty annoying.

**Comment by ghost** [comment]
I have contentkeeper, really annoying since it's not an extension

**Reply by Luigi-Master** [reply]
So would it be in your managed network?

**Reply by ghost** [reply]
yeah sort of but it still blocks at home, with a home computer its blocks on wifi, but it blocks somehow with managed chromebooks, theres another extension called contentkeeper auth but it just checks to see if you´re account is allowed on that website, like my brother can´t use youtube but I can

**Reply by schoolexploitkid** [reply]
use a home computer, and then use proton vpn. thats what i do on my mac

**Comment by Ltbeef** [comment]
I got blocksi

**Comment by Dario-git-hub** [comment]
I have iRiS school
